source/ceylon/toml/formatToml.ceylon

```diff
@@ -87,6 +87,17 @@ shared String formatToml(Map<String, Object> tomlTable)
         emitBasicString(item);
     }
 
+    "Is the character in the Unicode 'private use' category?"
+    function privateUse(Character char)
+        =>     '\{#E000}' <= char <= '\{#F8FF}'
+            || '\{#0F0000}' <= char <= '\{#0FFFFD}'
+            || '\{#100000}' <= char <= '\{#10FFFD}';
+
+    "Is the character in the Unicode 'control' category?"
+    function control(Character char)
```

Comment:
I actually wrote it that way first :)

It's subjective, but I didn't like 1) how `Character.control` was documented, 2) that `Character.control` additionally delegates to Java's impl, and 3) the asymmetry with the other helper.